src/main/java/core/basesyntax/EntryToFile.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class EntryToFile {
+    public void writeToFile(Report report, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write("supply," + report.getSupplies() + System.lineSeparator());
```

Comment:
Let's create constants for supply, buy, result and comma. 
Also, it's better to prepare String report by using StringBuilder and avoid concatenation and write it to file